styleguide/_static/issue-templates/security.md

```diff
@@ -0,0 +1,58 @@
+---
+name: 🔐 Security Issue
+about: Fix Security Issue
+title: 🔐 <Insert Title>
+labels: security
+assignees: ''
+
+---
+
+# Summary 
+<!--- 
+A short summary of the security issue.
+--->
+
+|            |     |
+|------------|-----|
+| Class/Type | -   |
+| Version(s) | -   |
+| Severity   | -   |
+| CVE Number | -   |
+
+<!---
+A quick table based overview, containing the most important details.
```

Comment:
What does this table focus on, on security issues in the project itself, or is it also meant represent security issues in the dependencies.